iconic/accounts/urls.py

```diff
@@ -6,4 +6,5 @@
     path('data/', views.UserRetrieveUpdateDestroyAPIView.as_view(),
          name='user-data'),
     path('users/', include(routers.router.urls)),
+    path('users/', include('UserProfile.urls'))
```

Comment:
Not there, this line should be in iconic/iconic/urls.py

Reply:
you make me sad